dss-api.yml

```diff
@@ -119,11 +119,12 @@ securityDefinitions:
       - '{{OIDC_EMAIL_CLAIM}}': the email of the requester
     type: oauth2
     flow: implicit
-    authorizationUrl: https://auth.data.humancellatlas.org/authorize
+    authorizationUrl: https://auth.ucsc.ucsc-cgp-redwood.org/authorize
```

Comment:
You use `{{API_DOMAIN_NAME}}` below, should this also use an environment variable for the authorization URL?

Reply:
its going to be a fixed value (most of the time), but i agree lets see if this can be extracted from env var.

Reply:
it works!~